File: javascript/1295/Solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    return nums.map(num => num.toString())
        .map(string => string.length)
        .filter(length => length % 2 == 0)
        .length;
};

File: java/1295/Solution.java
import java.util.stream.IntStream;

class Solution {
    public int findNumbers(int[] nums) {
        return (int) IntStream.of(nums)
            .mapToObj(Integer::toString)
            .map(String::length)
            .filter(length -> length % 2 == 0)
            .count();
    }
}
